shaharyar/README.md

```diff
@@ -0,0 +1,3 @@
+# How to run the code
+- pip install -r requirements.txt
+- cdk synth && cdk deploy
```

Comment:
I will encourage you to add only one Readme file rather than readme and readme1 in for the same folder